README.md

```diff
@@ -14,4 +42,104 @@ export const CompareText = () => {
   )
 }
 ```
+### To use `useTextComparison` hook
+```js
+import { useTextComparison } from 'text-compare';
+
+const TextDiffer = () => {
+  const text1 = 'The quick brown fox jumps over the lazy dog';
+  const text2 = 'The quick blue fox leaps over the happy dog';
+
+  const { comparisonResult, similarity } = useTextComparison(text1, text2);
+
+  return (
+    <>
+      <div className="flex gap-1 mb-4">{comparisonResult}</div>
+      <div>Similarity: {similarity.toFixed(2)}%</div>
+    </>
+  );
+};
+```
+
+## Advanced Usage 🔧
+
+### Customizing Colors
+
+The `useTextComparison` hook accepts custom colors for highlighting differences:
+
+```jsx
+const customColors = {
+  commonColor: '#1E90FF',    // DodgerBlue for common words
+  removedColor: '#FF6347',   // TomatoRed for removed words
+  addedColor: '#32CD32',     // LimeGreen for added words
+};
+
+const { comparisonResult, similarity } = useTextComparison(text1, text2, {
+  customColors,
+});
+```
+
+### Available Options
+
+| Option | Default | Description |
+|--------|---------|-------------|
+| `commonColor` | "gray" | Color for words that appear in both texts |
+| `removedColor` | "red" | Color for words that only appear in the first text |
+| `addedColor` | "green" | Color for words that only appear in the second text |
```

Comment:
Should be only one row that talk about a customColor object with three properties. You should be explaining the options object here

Reply:
Taken care as per comment.